nav2_map_server/CMakeLists.txt

```diff
@@ -1,45 +1,40 @@
 cmake_minimum_required(VERSION 3.5)
 project(nav2_map_server)
 
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake_modules)
+
 find_package(ament_cmake REQUIRED)
 find_package(nav2_common REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_lifecycle REQUIRED)
 find_package(nav_msgs REQUIRED)
-find_package(SDL REQUIRED)
-find_package(SDL_image REQUIRED)
 find_package(yaml_cpp_vendor REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(nav2_util REQUIRED)
+find_package(GRAPHICSMAGICKCPP REQUIRED)
```

Comment:
Are there / have you added rosdep keys for this?

Reply:
Already exists :-)

Reply:
![nice](https://media1.tenor.com/images/d2c70f7f64587dc3b7c86ee06756fb4a/tenor.gif?itemid=4294979)